zebra-scan/src/scan.rs

```diff
@@ -68,17 +68,17 @@ pub async fn start(
 
```

Comment:
We're reading all the blocks even if there isn't a key that needs them. So let's set `height` to the minimum height in `key_heights` instead?

Reply:
https://github.com/ZcashFoundation/zebra/pull/8080/commits/0a5c08628abb412515acc4bd6c3b10ad3be3dc34